app/models/grda_warehouse/client_file.rb

```diff
@@ -41,7 +41,7 @@ class ClientFile < GrdaWarehouse::File
       #   show those with full releases and those you uploaded
       elsif user.can_manage_window_client_files?
         window.where(
-          arel_table[:client_id].in(Arel.sql(GrdaWarehouse::Hud::Client.full_housing_release_on_file.select(:id).to_sql)).
+          arel_table[:client_id].in(Arel.sql(GrdaWarehouse::Hud::Client.full_housing_release_on_file(coc_codes: user.coc_codes_for_consent).select(:id).to_sql)).
```

Comment:
The CoC Codes need to come from the uploaded file not the user.  See `GrdaWarehouse::ClientFile#consent_type`.  At the moment this just shows one CoC code, so let's implement it that way.  Eventually we'll need to change the way those are collected/stored, so we'll need to show multiple.